rollup.config.js

```diff
@@ -0,0 +1,29 @@
+import babel from 'rollup-plugin-babel'
+import cjs from 'rollup-plugin-commonjs'
+import globals from 'rollup-plugin-node-globals'
+import replace from 'rollup-plugin-replace'
+import resolve from 'rollup-plugin-node-resolve'
+
+export default {
+  entry: `src/${process.env.entry}.js`,
+  dest: `dist/${process.env.entry}.js`,  
+  format: 'iife',
+  plugins: [
+    babel({
+      babelrc: false,
+      exclude: 'node_modules/**',
+      presets: [ [ 'es2015', { modules: false } ], 'stage-0', 'react' ],
+      plugins: [ 'external-helpers' ]
+    }),
+    cjs({
+      exclude: 'node_modules/process-es6/**',
+    }),
+    globals(),
+    replace({ 'process.env.NODE_ENV': '"development"' }),
```

Comment:
Kind of an in-case-we-need-it thing that I copypasta'd from the rollup example I found while setting this all up. We could enable debugging stuff & things like a super-short snooze time for a dev build - and then turn that stuff off in an automated prod build.